tutormfe/templates/mfe/build/mfe/Dockerfile

```diff
@@ -51,43 +51,24 @@ RUN npm clean-install --no-audit --no-fund --registry=$NPM_REGISTRY \
 {{ patch("mfe-dockerfile-post-npm-install") }}
 COPY --from={{ app["name"] }}-src /openedx/app /openedx/app
 COPY --from={{ app["name"] }}-i18n /openedx/app/src/i18n/messages /openedx/app/src/i18n/messages
-ENV PUBLIC_PATH='/{{ app["name"] }}/'
 EXPOSE {{ app['port'] }}
 
-COPY ./env/production /openedx/env/production
-{%- set overrides = app.get("env", {}).get("production", {}) %}
-{%- if overrides %}
-RUN echo "setting production overrides..." \
-  {%- for key, value in app.get("env", {}).get("production", {}).items() %}
-  && echo "{{ key }}='{{ value }}'" >> /openedx/env/production \
-  {%- endfor %}
-  && echo "done setting production overrides"
-{%- endif %}
+# configuration needed at build time 
+ENV APP_ID={{ app["name"] }}
+ENV PUBLIC_PATH='/{{ app["name"] }}/'
+ENV MFE_CONFIG_API_URL=/api/mfe_config/v1
+ENV ENABLE_NEW_RELIC=false
```

Comment:
Why do we refer to New Relic specifically?  Is this variable ever true by default?

Reply:
I though you were mentioning as need at build time in previous comment?

Reply:
Yeah, but since it's false by default and I don't expect people to actually use it, I don't think we need to worry about it here.  Best to remove it.